hugo/layouts/index.html

```diff
@@ -61,169 +34,238 @@ <h4 class="text-dark m-0">
     {{ end }}
 </section>
 
-<section id="why" class="why pt-5 pb-0 pb-m-5">
+<section id="engineering" class="engineering pt-5 pb-0 pb-m-5">
+    <div class="container">
+        <header class="text-center pb-3">
+            <h2 class="text-white">Engineering Observability is the key to succesfull IT initiatives</h2>
+        </header>
+        <div class="row">
+            <ul class="nav nav-tabs eng-tabs col-12 text-center mb-3 mb-md-0" id="engTabs" role="tablist">
+                <li class="nav-item eng-tab-1 col-sm-12 col-md-3">
```

Comment:
clicking in these tabs (one after the other), causes a weird scrolling of the page
![image](https://user-images.githubusercontent.com/2437584/58095442-4ce67b80-7bd3-11e9-9cf7-bfdecdc7fae1.png)